tools/CoherenceBuild/CoherenceBuild.cs

```diff
@@ -81,13 +81,16 @@ private void ProcessRepo(ConcurrentBag<PackageInfo> processedPackages, Repositor
             repoDirectory = Path.Combine(repoDirectory, repo.BuildNumber);
 
             var buildDirectory = Path.Combine(repoDirectory, repo.BuildDirectory);
+            var lineupPackageDirectory = Path.Combine(repoDirectory, repo.LineupPackageDirectory);
```

Comment:
aspnet/Universe creates Internal.AspNetCore.Universe.Lineup and aspnet/CoreCLR creates Internal.AspNetCore.Partners.Lineup